### ISSUE TITLE
Support specifying dynamic quality for lossy images through query parameter (q)

### ISSUE THREAD

**Issue by adalberht** [gojek/darkroom#30]
##### What would you like to be added
`q` query parameters support in Manipulator that enables dynamic quality based on query

##### Why is this needed
Currently all lossy images (JPEG & WebP) that is served by the same Processor will have the same quality, it would be nice to be able to specify `q` query parameter

**Comment by baskarap** [comment]
Is `q` the compression rate like what we have in internal darkroom at Gojek?

**Comment by adalberht** [comment]
@baskarap yes you are right, currently we support the `q` compression rate at Manipulator's constructor level, which results in constructing new Manipulator object for handling different `q` compression rate requests
